engine/src/snapshot/migration/v3.rs

```diff
@@ -0,0 +1,57 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use super::*;
+
+/// Key size for the ephemeral key
+const KEY_SIZE: usize = 32;
+/// Key type alias.
+pub type Key = [u8; KEY_SIZE];
+
+const VERSION_V3: [u8; 2] = [0x3, 0x0];
+
+#[allow(dead_code)]
+pub(crate) fn read<I: Read>(input: &mut I, password: &Key, _associated_data: &[u8]) -> Result<Vec<u8>, Error> {
+    let mut age = Vec::new();
+    input.read_to_end(&mut age)?;
+    age::decrypt_vec(password, age::RECOMMENDED_MAXIMUM_DECRYPT_WORK_FACTOR, &age[..]).map_err(From::from)
+}
+
+pub(crate) fn write<O: Write>(
+    plain: &[u8],
+    output: &mut O,
+    password: &Key,
+    _associated_data: &[u8],
```

Comment:
Deliberately ignored variable? If so, can you add a comment why?

Reply:
No, just forgot to remove it. My bad. Fixed it.